python/cudf/cudf/pandas/_wrappers/pandas.py

```diff
@@ -891,6 +891,28 @@ def Index__new__(cls, *args, **kwargs):
 
 _eval_func = _FunctionProxy(_Unusable(), pd.eval)
 
+_read_pickle_func = _FunctionProxy(_Unusable(), pd.read_pickle)
+
+_to_pickle_func = _FunctionProxy(_Unusable(), pd.to_pickle)
```

Comment:
I think we can simplify these definitions below and just write:

```
register_proxy_func(pd.read_pickle)(_FunctionProxy(_Unusable(), pd.read_pickle))
register_proxy_func(pd.to_pickle)(_FunctionProxy(_Unusable(), pd.to_pickle))
```

Then we are immune to changes in the arguments especially to `read_pickle`